src/Database/Database.php

```
@@ -6578,11 +6658,11 @@ public function encode(Document $collection, Document $document): Document
      *
      * @param Document $collection
      * @param Document $document
-     * @param array<string> $selections
+     * @param array<Query> $selects
      * @return Document
      * @throws DatabaseException
      */
-    public function decode(Document $collection, Document $document, array $selections = []): Document
+    public function decodeOriginal(Document $collection, Document $document, array $selects = []): Document
```

PHPMD (2.15.0)

Avoid unused parameters such as '$selects'. (undefined)

(UnusedFormalParameter)